src/views/Game/PlayerCards.tsx

```diff
@@ -352,13 +356,18 @@ export function PlayerCard({
                         })}
                     </div>
                 )}
-                <NumCapturesText
-                    score={score}
-                    color={color}
-                    zen_mode={zen_mode}
-                    hidden={show_points && !estimating_score}
-                />
-                {!show_points && <div className="komi">{komiString(score.komi)}</div>}
+                {(!show_points || estimating_score) && (
```

Comment:
This is a drive-by fix, where the captures text would sometimes take up extra space when it was hidden. Instead, don't call `NumCapturesText` at all.